playbooks/group_vars/all/global.yml

```diff
@@ -1,108 +1,34 @@
 ---
 # run roles based on certain params
+skip_clean: false
```

Comment:
IIUC it is best practice to use idempotent way in ansible. But default for `skip_clean` is false.
Could you explain the reason?

Reply:
if do you use this flag as true can you destroy your environment and return to scratch but after installed the first time do you can run another role separated without clean because this exist this option.